react.js

```diff
@@ -18,6 +32,13 @@ for(let i=0; i<names.length; i++){
 }
 
 
+//MY CHANGES:
+
+names.map((v,i) => {
+    console.log(`${names[i]} is ${names[i].length} characters long.`)
+})
+
```

Comment:
${names[i]} can also be ${v} - it works both ways but since you have value you might as well use it.